app/assets/stylesheets/application.scss

```diff
@@ -15,28 +15,74 @@
 *= require foundation_and_overrides
 *= require_self
  */
+@import url(https://fonts.googleapis.com/css?family=Montaga);
+$lightgray: #ede0d5;
+$blue: #354a57;
 $borderbrown: #a52a2a;
 $backgroundorange: #ffffcc;
 $fontgreen: #008000;
+$fontblue: #94c9ff;
 
-.side-nav {
-  height: 100vh;
+
+.content {
+  padding-left: 20px;
+}
+.side-nav  {
+  background-color: #b94947;
+  padding: 1em;
+  color:$blue !important ;
+  h1, h2, h3, h4, h5 {
+    color:$blue !important ;
```

Comment:
!important should not be used<br>Each selector in a comma sequence should be on its own line<br>Colon after property should be followed by one space<br>Declaration should not have a space before the terminating semicolon